frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -21,418 +21,444 @@ const { ORDERS, ORDERS_ID, PEOPLE, PEOPLE_ID, PRODUCTS, PRODUCTS_ID } =
-    join();
-    popover().findByText("Orders model").click();
-    popover().findByText("ID").click();
-    popover().findByText("Product ID").click();
+    // flaky test
+    it.skip("should show an info popover when hovering over a field picker option for a table", () => {
+      startNewQuestion();
+      cy.contains("Sample Database").click();
+      cy.contains("Orders").click();
 
-    visualize();
-  });
+      cy.findByTestId("fields-picker").click();
 
-  // flaky test
-  it.skip("should show an info popover when hovering over a field picker option for a table", () => {
-    startNewQuestion();
-    cy.contains("Sample Database").click();
-    cy.contains("Orders").click();
+      cy.findByText("Total").trigger("mouseenter");
 
-    cy.findByTestId("fields-picker").click();
+      popover().contains("The total billed amount.");
+      popover().contains("80.36");
+    });
 
-    cy.findByText("Total").trigger("mouseenter");
+    // flaky test
+    it.skip("should show an info popover when hovering over a field picker option for a saved question", () => {
+      cy.createNativeQuestion({
+        name: "question a",
+        native: { query: "select 'foo' as a_column" },
+      });
 
-    popover().contains("The total billed amount.");
-    popover().contains("80.36");
+      // start a custom question with question a
+      startNewQuestion();
+      cy.findByText("Saved Questions").click();
+      cy.findByText("question a").click();
+
+      cy.findByTestId("fields-picker").click();
+
+      cy.findByText("A_COLUMN").trigger("mouseenter");
+
+      popover().contains("A_COLUMN");
+      popover().contains("No description");
+    });
   });
 
-  // flaky test
-  it.skip("should show an info popover when hovering over a field picker option for a saved question", () => {
-    cy.createNativeQuestion({
-      name: "question a",
-      native: { query: "select 'foo' as a_column" },
+  describe("postgres", () => {
+    beforeEach(() => {
+      restore("postgres-12");
+      cy.signInAsAdmin();
     });
 
-    // start a custom question with question a
-    startNewQuestion();
-    cy.findByText("Saved Questions").click();
-    cy.findByText("question a").click();
+    it('should show Median aggregation function for databases that support "percentile-aggregations" driver feature', () => {
+      // start a custom question with orders
+      startNewQuestion();
+      cy.contains("QA Postgres12").click();
+      cy.contains("Products").click();
 
-    cy.findByTestId("fields-picker").click();
+      cy.contains("Summarize").click();
 
-    cy.findByText("A_COLUMN").trigger("mouseenter");
+      cy.contains("Pick the metric you want to see").click();
+      popover().within(() => {
+        cy.findByText("Median of ...").click();
+        cy.findByText("Price").click();
+      });
 
-    popover().contains("A_COLUMN");
-    popover().contains("No description");
+      cy.contains("Median of Price");
+    });
```

Comment:
Let's also call `visualize();` and assert on results. There should be a median price of all products.